watertap/data/techno_economic/municipal_drinking.yaml

```
@@ -1,8 +1,14 @@
-default:
-  recovery_frac_mass_H2O:
-    value: 0.9999
-    units: dimensionless
-    reference: .nan
-  default_removal_frac_mass_solute:
-    value: 0
+default:  # needed some parameters as placeholder, so including cost params here
+  cap_basis_par:
+    value: 0.0403
+    units: M$       # TODO: unsure of the units
```

```suggestion
    units: M$
```
I think we resolved this, and `M$` is correct. @kurbansitterley thoughts?

---

@kurbansitterley  confirmed that these units are correct.

---

As a note, this will have to change anyway once the costing code is up - I hope to have a draft up today which will show the proposed format. Thus I would suggest we not touch this for now.